.pipelines/ci.yml

```diff
@@ -36,161 +36,254 @@ variables:
+              summaryFileLocation: $(System.DefaultWorkingDirectory)/**/coverage.xml
+              failIfCoverageEmpty: false
+            condition: succeededOrFailed()
+            target: golang
+
+      - job: Lint_Admin_Portal
+        pool:
+          name: 1es-aro-ci-pool
+        steps:
+          - template: ./templates/template-checkout.yml
+          - script: |
+              set -xe
+              make lint-admin-portal
+            displayName: 🧹 Lint Admin Portal
+
+  - stage: Containerized
+    dependsOn: []
+    jobs:
+      - job: Build_Test_And_Push_Az_ARO_Extension
+        pool:
+          name: 1es-aro-ci-pool
+
+        variables:
+          ACR_FQDN: 'arosvcdev.azurecr.io'  # ACR FQDN
+          REPOSITORY: 'my-image-repo'
+          IMAGE_TAG: $(Build.SourceBranchName)
+        steps:
+          - template: ./templates/template-checkout.yml
+
+          - task: AzureCLI@2
+            inputs:
+              azureSubscription: 'ado-pipeline-dev-image-push'  # Service connection name
+              scriptType: 'bash'
+              scriptLocation: 'inlineScript'
+              inlineScript: |
+                # Log in to ACR and get the token
+                echo "Logging in to Azure and retrieving ACR token..."
+                TOKEN=$(az acr login --name ${ACR_FQDN%%.*} --expose-token --output tsv --query accessToken)
+
+                if [ -z "$TOKEN" ]; then
+                  echo "Error: Failed to retrieve ACR token."
+                  exit 1
+                fi
+
+                # Login to ACR using Podman
+                echo "Logging into ACR with Podman..."
+                echo $TOKEN | podman login ${ACR_FQDN} -u "00000000-0000-0000-0000-000000000000" --password-stdin --cgroup-manager=cgroupfs 2>/dev/null
+
+                # Check if login was successful
+                if [ $? -ne 0 ]; then
+                  echo "Error: Podman login failed."
+                  exit 1
+                fi
+                echo "Podman login successful."
+
+          - script: |
+              set -xe
+              export NO_CACHE=true
+
+              # Install podman
+              sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+              sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+              sudo dnf install -y https://packages.microsoft.com/config/rhel/8/packages-microsoft-prod.rpm
+              sudo dnf install -y azure-cli
+              sudo dnf install -y podman make
+              which podman
+              podman --version
+              whoami
+
+              # Source the podman control script
+              . ./hack/e2e/utils.sh
+              run_podman
+              validate_podman_running
+
+              # Run the build and capture the image tag
+              export PODMAN_REMOTE_ARGS="-r --url=tcp://127.0.0.1:8888"
+              IMAGE_TAG=$(make azext-aro | grep 'Successfully tagged' | awk '{print $3}' | cut -d':' -f2)
```

Comment:
idea: instead of using `awk` on a log to get an image tag (and thus also hiding the log from view in the build), then retagging that image, we should be able to save both steps, and just override `LOCAL_AZ_EXT_ARO_IMAGE` envvar to set it to the image name we need, right? If our issue is that we need to know what `VERSION` is, let's just make a Makefile target like `make version` to grab that data directly rather than parsing a log.